game/src/backend/constants.rs

```diff
@@ -47,10 +47,11 @@ pub(crate) const SANDSTURM_CR: Resources<i16> = Resources {
 #[allow(clippy::too_many_lines)]
 /// Generates all machines with all their name, position, trades and resources.
 /// # Returns
-/// `String` - The name of the machine.
-/// `Rect` - Returns the collision area of the machine.
-/// `Vec<Trade>` - Returns the trades of the machine.
-/// `Vec<Resources>` - Returns the resources of the machine.
+/// A Vector of Tuples containing the following values
```

Comment:
Vector of Machines

Reply:
ist es aber nicht

Reply:
doch, hab ich so geändert, weil die Funktion einfach sehr useless war, da man auf jedes Element dann eh nur new_from_const angewandt hat